python/ray/autoscaler/_private/commands.py

```diff
@@ -90,7 +95,8 @@ def debug_status():
     return status
 
 
-def request_resources(num_cpus=None, bundles=None):
+def request_resources(num_cpus: Optional[int] = None,
+                      bundles: List[Dict] = None) -> None:
```

Comment:
should this be an `Optional[List[Dict]]`?

Reply:
Yep.